### ISSUE TITLE
Add toggle in settings to default to unlisted for replies

### ISSUE THREAD

**Issue by libertyforward1** [enafore/enafore#26]
Suggestion: Adding a toggle in the settings menu to default replies to "unlisted" visibility. This could then be changed manually as needed.

Thanks for the hard work. I'm so glad to see Pinafore reincarnated.

**Comment by easrng** [comment]
You want replies to not follow your account's default post visibility settings?

**Comment by libertyforward1** [comment]
> You want replies to not follow your account's default post visibility settings?

Correct. I'm fine with my new posts being public, but usually (not *always*, just usually) prefer replies to automatically be unlisted; occasionally I go on a posting spree and *do* want to have them visible in the public timeline, hence a toggle for such occasions.

**Comment by Domoel** [comment]
My fork does support this feature.

https://zocial.social
